src/modules/createRandomNumber.js

```diff
@@ -0,0 +1,15 @@
+const createRandomNumber = () => {
```

Comment:
here you allow to create number that start from 0

Reply:
what is meant? 0123 is this number not correct?

Reply:
no, number shouldn't start with zero

Reply:
okey i change it. thx